### PR TITLE
Feature/converters

### PR DESCRIPTION
**Pull request by kurianov-pavel**

### CODE REVIEW

File: XamBasePacket/Converters/IntToBoolConverter.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace XamBasePacket.Converters
+{
+	public class IntToBoolConverter : IValueConverter
```
Comment (stepkillah) [inline XamBasePacket/Converters/IntToBoolConverter.cs]: Better to find a better name than generic `IntToBool`. In this particular case we comparing only greater then zero cases, we could add an equal zero and less than zero, or more complex int to bool converters in the future.

File: XamBasePacket/Converters/BoolToStringConverter.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace XamBasePacket.Converters
+{
+    public class BoolToStringConverter : IValueConverter
+    {
+        public string TrueValue { get; set; }
```
Comment (stepkillah) [inline XamBasePacket/Converters/BoolToStringConverter.cs:10]: Add `Null` option

File: XamBasePacket/Converters/NullToBoolConverter.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+namespace XamBasePacket.Converters
+{
+    public class NullToBoolConverter : IValueConverter
+    {
+        public bool WhenNull { get; set; }
+
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value is string str)
```
Comment (stepkillah) [inline XamBasePacket/Converters/NullToBoolConverter.cs]: I think we might want to add one more clause for collections, something like
`if(value is IList/ICollection collection)
return collection == null || !collection.any() ? WhenNull : !WhenNull`

**Review by stepkillah — CHANGES_REQUESTED** [review]

File: XamBasePacket/Converters/IntToBoolConverter.cs (hunk 1 above)
Comment (kurianov-pavel) [reply XamBasePacket/Converters/IntToBoolConverter.cs]: IntToBool - in the future, it needs full refactor for all other generic types

File: XamBasePacket/Converters/BoolToStringConverter.cs (hunk 2 above)
Comment (kurianov-pavel) [reply XamBasePacket/Converters/BoolToStringConverter.cs:10]: Added

File: XamBasePacket/Converters/NullToBoolConverter.cs (hunk 3 above)
Comment (kurianov-pavel) [reply XamBasePacket/Converters/NullToBoolConverter.cs]: Added

**Review by stepkillah — APPROVED** [review]

**Pull request merged**